src/docs/asciidoc/reference/manage/organization.adoc

```diff
@@ -0,0 +1,36 @@
+:relative-path: ../../../
+include::{docdir}/variables.adoc[]
+
+An *organization* is a concept to let you work in team.
+
+The organization commonly groups all *ZetaPush accounts* of your company.
+You can have several applications for your organization and you can manage access rights on applications according to members of your team.
```

Comment:
We should explain what is an application before using it even if it seems obvious to us